client/reader/following-manage/index.jsx

```diff
@@ -43,6 +43,29 @@ import { recordTrack } from 'reader/stats';
 
 const PAGE_SIZE = 4;
 
+function reportOnPropChange( Wrapped, prop ) {
```

Comment:
we can get rid of this whole thing since its driven by a prop from following-manage/index!